tox.ini

```diff
@@ -19,8 +19,19 @@ commands =
 
 [testenv]
 deps =
-    -rrequirements_test.txt
+    pytest>=3.0
     dj18: Django>=1.8,<1.9
     dj19: Django>=1.9,<1.10
     dj110: Django>=1.10,<1.11
+    postgres: psycopg2>=2.6
```

Comment:
Wouldn't it be safer to pin these requirements on major version numbers, say ``psycopg2>=2.6,<2.7``?
Sometimes it happened that we receive a pull request which works in theory, but because of some API changes in the underlying dependency the unit tests fail.